handlers/gwstreamer.py

```diff
@@ -277,6 +277,7 @@ def handle(self, gcn_alert: dict) -> None:
         
         with open(f'{self.OUTPUT_TRIGGER}/{trigger_id}.json', 'w') as jsonfile:
             json.dump(record, jsonfile)
+            os.chmod(jsonfile, 0o0777)
```

Comment:
I think here instead of os.chmod(jsonfile, 0o0777) should be the os.chmod(f'{self.OUTPUT_TRIGGER}/{trigger_id}.json', 0o0777) (line 280)

Reply:
Updated - good catch!